Assets/_BuildingBlocks/Credentials/Scripts/StoredAuthorization/UsernamePassword.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using UnityEngine.Networking;
+
+namespace Netherlands3D.Credentials.StoredAuthorization
+{
+    [Serializable]
+    public class UsernamePassword : HeaderBasedAuthorization
+    {
+        public string username;
+        public string password => key;
+        public override string headerPrefix { get; protected set; } = "Basic ";
+        public override AuthorizationType AuthorizationType => AuthorizationType.UsernamePassword;
+
+        public UsernamePassword(Uri url, string username, string password) : base(url, password)
+        {
+            this.username = username;
+        }
+        
+        
+        public UnityWebRequest GetWebRequestWithHeader()
+        {
+            var uwr = new UnityWebRequest();
+            uwr.SetRequestHeader(headerName, headerPrefix + GetUserNamePassWordQuery());
+            return uwr;
+        }
+
+        public string GetUserNamePassWordQuery()
```

Comment:
`GetUserNamePassWordQuery` -> `GetUsenNamePasswordQuery` (username en password zijn 1 woord)

Reply:
Deze zou niet public moeten zijn, dit is een afhandeling van de interne staat